wallet/wallet-rpc-lib/src/service/worker.rs

```diff
@@ -209,6 +248,10 @@ impl WalletWorker {
         Ok(result)
     }
 
+    fn broadcast(&mut self, event: &Event) {
+        self.subscribers.retain(|sub| sub.send(event.clone()).is_ok())
```

Comment:
I think this is a little too aggressive, removing subscribers just because there's an error. Perhaps there's a better way to handle errors. Off the top of my head, we should allow unsubscribing and/or clearing subscribers, and when a failure occurs, we write a log message.

In practice, an absent subscriber is the subscriber's problem, not the service's problem. Those who care can always add more subscribers, who cares about dead subscribers? We're only limited by memory and a slight dent in performance.

Reply:
One important point here is to make sending notifications async. Given that websocket uses TCP, if one subscriber doesn't respond, this transaction will take a very long time to complete, blocking other subscribers. This glosses on the issue of the above comment. It shouldn't make a difference if a subscriber is dead/absent or whatever prevents the notifications from arriving and it shouldn't have an effect on others, to the extent possible by the async library.

Reply:
If async isn't an option due to time constraints, going as far as running a thread pool isn't wouldn't be super-crazy... at least until we have the time to do this right.

Reply:
> I think this is a little too aggressive, removing subscribers just because there's an error. Perhaps there's a better way to handle errors. Off the top of my head, we should allow unsubscribing and/or clearing subscribers, and when a failure occurs, we write a log message.

This only happens when the other channel endpoint is already dead. There is an unsubscribe method generated by `jsonrpsee` so subscribers can clean up as they wish.